docs/migration_guide.md

```diff
@@ -1,36 +1,28 @@
-# Harbor upgrade and migration guide
+# Harbor Upgrade and Migration Guide
 
-This guide only covers upgrade and migration to version >= v1.8.0
+This guide covers upgrade and migration to versions >= v1.9.0. This guide only covers migration from v1.7.x and later to the current version. If you are upgrading from an earlier version, refer to the migration guide in the `release-1.7.0` branch to upgrade to v1.7.x first, then follow this guide to perform the migration to this version. 
```

Comment:
It's migration to versions v1.9.x correct?  1.10 might introduce different behavior, we don't know for sure yet.

Reply:
sounds good